docs/tutorials/multimodal/document/pdf_classification.md

```diff
@@ -0,0 +1,112 @@
+# Classifying PDF Documents with AutoMM
+PDF comes short from Portable Document Format and is one of the most popular document formats.
+We can find PDFs everywhere, from personal resumes to business contracts, and from commercial brochures to government documents. 
+The list can be endless. 
+PDF is highly praised for its portability. 
+There's no worry about the receiver being unable to view the document or see an imperfect version regardless of their operating system and device models.
+
+Using AutoMM, you can handle and build machine learning models on PDF documents just like working on other modalities such as text and images, without bothering about PDFs processing. 
+In this tutorial, we will introduce how to sort PDF documents automatically with AutoMM using document foundation models. Let’s get started!
+
+
+## Get the PDF document dataset
+We have created a simple PDFs dataset via manual crawling for demonstration purpose. 
+It consists of two categories, resume and historical (downloaded from [milestone documents](https://www.archives.gov/milestone-documents/list)). 
+We picked 20 PDF documents for each of the category. 
+
+Now, let's download the dataset and split it into training and test sets.
+
+```{.python .input}
+import warnings
+warnings.filterwarnings('ignore')
+import os
+import pandas as pd
+from autogluon.core.utils.loaders import load_zip
+
+download_dir = './ag_automm_tutorial_pdf_classifier'
+zip_file = "https://automl-mm-bench.s3.amazonaws.com/doc_classification/pdf_docs_small.zip"
+load_zip.unzip(zip_file, unzip_dir=download_dir)
+
+dataset_path = os.path.join(download_dir, "pdf_docs_small")
+pdf_docs = pd.read_csv(f"{dataset_path}/data.csv")
+train_data = pdf_docs.sample(frac=0.8, random_state=200)
+test_data = pdf_docs.drop(train_data.index)
+```
+
+Now, let's visualize one of the PDF documents. Here, we use the S3 URL of the PDF document and `IFrame` to show it in the tutorial.
+```{.python .input}
+from IPython.display import IFrame
+
+IFrame("https://automl-mm-bench.s3.amazonaws.com/doc_classification/historical_1.pdf", width=400, height=500)
```

Comment:
Since the pdfs are downloaded to local, why don't visualize a pdf after expanding the paths below?

Reply:
IFrame cannot visualize pictures from other folders.

Reply:
Do you mean `IFrame("a/local/path/to/file.pdf", width=400, height=500)` doesn't work?

Reply:
Yeah, it won’t work.